modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/ObjectEntryResourceTest.java

```diff
@@ -8532,6 +8544,84 @@ public void testPostRelatedObjectEntryInDifferentCompany()
 		);
 	}
 
+	@FeatureFlags("LPD-37056")
+	@Test
+	public void testPostScopeScopeKeyValidate() throws Exception {
+		ObjectField textObjectField = _objectFieldLocalService.getObjectField(
+			_siteScopedObjectDefinition1.getObjectDefinitionId(),
+			_OBJECT_FIELD_NAME_1);
+
+		String error = RandomTestUtil.randomString();
+
+		ObjectValidationRule objectValidationRule =
+			_objectValidationRuleLocalService.addObjectValidationRule(
+				"objectValidationRuleERC", TestPropsValues.getUserId(),
+				_siteScopedObjectDefinition1.getObjectDefinitionId(), true,
+				ObjectValidationRuleConstants.ENGINE_TYPE_DDM,
```

Comment:
It seems your tests only cover the scenario where the object definition has only one objectValidationRule using the ENGINE_TYPE_DDM. Can we elaborate further to also cover the scenario where the object definition has more than one ObjectValidationRule? including from different engine types?